src/kiutils/utils/sexpr.py

```diff
@@ -11,7 +11,7 @@
         (?P<brackl>\()|
         (?P<brackr>\))|
         (?P<num>[+-]?\d+\.\d+(?=[\ \)])|\-?\d+(?=[\ \)]))|
-        (?P<sq>"([^"]|(?<=\\)")*")|
+        (?P<sq>"([^"]|(?<=(?:\\\\|.[^\\])\\)")*")|
```

Comment:
As you said, this does only match your one problem case. I took a look into this and it seems like by just making the innter unnamed capture goup a none-capturing group, it parses every combination of \\\ and "\ just fine:
```
(?P<sq>"(?:[^"]|(?<=\\)")*")|
```

Something like this now works fine:
```   
(property "Fo|\\\\\\"\\"\"\\\\\\ot\"print" "some:part" (id 2) (at 106.68 683.895 0)
  (effects (font (size 1.27 1.27)) hide)
)
```

The official KiCad Library Utils also changes something here as well since i reused their S-Expression parser. It now does it the same way: https://gitlab.com/kicad/libraries/kicad-library-utils/-/blob/master/common/sexpr.py#L16

Please replace this line with the regex above and try parsing your problem file again if it still works as expected. Our unittests still work fine with this change on my local machine.